Cargo.toml

```diff
@@ -2,7 +2,10 @@
 name = "stabilizer"
 version = "0.5.0"
```

Comment:
Need to update this version as well

Reply:
Yes. I'll be using `cargo-release` for this which will do the increment in the end just before committing that and publishing.
Next time i'll look into splitting the sequence.

Reply:
In fact I'll try that workflow now.